src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -557,6 +574,7 @@ def set_input_embeddings(self, new_embeddings):
     def forward(
         self,
         input_ids: Optional[torch.LongTensor] = None,
+        attention_mask: Optional[torch.LongTensor] = None,
```

Comment:
this is breaking (having it as the second place)

Reply:
yes fixed it